### PR TITLE
docs(bodiless-cli, bodiless-accessibility): Bodiless Starter - `new` CLI command

### LINKED ISSUE #1594: [DOC] Bodiless Starter - "new" CLI command
## User Story

AS A Site Builder,
I WANT to know how to use the Bodiless CLI `new` command,
SO THAT I can create a new Bodiless site.

- AS A [who are you? eg Developer, Content Editor, etc.]
- I WANT [what is the feature you want?]
- SO THAT [what will you be able to do once the feature is implemented?]

### Acceptance Criteria

- Bodiless CLI `new` command is documented.

### References

- #1520

### PR DESCRIPTION
**Pull request by misterjones**
### `bodiless-cli`

* Add `doc/` directory, and move README file into it.
  * Update `package.json` and `bodiless.docs.json` accordingly.
* Clean up Bodiless CLI documentation.
* Update Bodiless CLI documentation with `new` and `new-vds` commands.

### `bodiless-accessibility`

* Add `doc/` directory, and move README file into it.
  * Update `package.json` and `bodiless.docs.json` accordingly.
* Clean up Bodiless Accessibility documentation.

### `bodiless-documentation`

* Clean up "Getting Started" guide.

### Documents:

* #1520 

---

Resolves #1594

### CODE REVIEW

File: packages/bodiless-documentation/doc/About/GettingStarted.md [hunk 1]
```diff
@@ -3,123 +3,137 @@
...
 npx @bodiless/cli new
 ```
 
-This will walk you through the process of creating a new bodiless site locally.  It will prompt you for the name of the new site, the path
-to the directory in which you want to create it, and the starter
-template you wish to use.  The following templates are available:
+This will walk you through the process of creating a new Bodiless site locally. It will prompt you
+for:
+
+- The revision of the source monorepo on which the new site will be based;
```
Comment (wodenx) [inline packages/bodiless-documentation/doc/About/GettingStarted.md:23]: This seems not to be true if you don't use the `-i` flag.  It uses the "latest" revision by default (the last tagged versino which is not a pre-release).  I think this is fine - but I would add here a note that, until the 1.0.0 release, you shoudl do:
```
npx @bodiless/cli@1.0.0-beta.11 new -r next
```
We can remove that as part fo the 1.0 release.

File: packages/bodiless-documentation/doc/About/GettingStarted.md [hunk 2]
```diff
@@ -3,123 +3,137 @@
...
-```
+```shell-session
 npm run build
 npm run serve
 ```
 
-Visit http://localhost:9000/ in your browser to view the site.
-
+Visit [http://localhost:9000/](http://localhost:9000/ ':target=_blank') in your browser to view the
+site.
```
Comment (wodenx) [inline packages/bodiless-documentation/doc/About/GettingStarted.md:55]: I think I"d add a paragraph here saying "To learn the basics of site guilding on bodiless, proceed to our tutorial, with link to the basic concepts doc)..  Or, read on to learn about developing for and contributing to the Bodiless project itself.

Comment (wodenx) [reply packages/bodiless-documentation/doc/About/GettingStarted.md:55]: Actualy - I think I'd put the "Exploring and Developing BodilessJS" in its own page.

Comment (wodenx) [reply packages/bodiless-documentation/doc/About/GettingStarted.md:55]: Or just move it below the "Troubleshooting" section.

File: packages/bodiless-documentation/doc/About/GettingStarted.md (hunk 1 above)
Comment (misterjones) [reply packages/bodiless-documentation/doc/About/GettingStarted.md]: ✅  Added note about executing `run` command on pre-release version.
✅  Moved info about choosing revision to its own section.

File: packages/bodiless-documentation/doc/About/GettingStarted.md (hunk 2 above)
Comment (misterjones) [reply packages/bodiless-documentation/doc/About/GettingStarted.md:72]: ✅  Added link to Basic Concepts tutorial.
✅  Moved "Exploring and Developing BodilessJS" section out to its own page.

**Review by hvanyo — DISMISSED** [review]

**Review by hvanyo — DISMISSED** [review]

File: packages/bodiless-documentation/doc/About/GettingStarted.md [hunk 3]
```diff
@@ -17,10 +17,27 @@ BodilessJS provides a CLI tool for creating a new site from a template.
...
+  npx @bodiless/cli@1.0.0-PRERELEASE.IDENTIFIER new -r next
+  ```
+
+  Where `PRERELEASE.IDENTIFIER` is the package version of `@bodiless/cli` that you are currently
+  using. E.g.:
+
+  ```shell-session
+  npx @bodiless/cli@1.0.0-beta.12 new -r next
+  ```
+
```
Comment (wodenx) [inline packages/bodiless-documentation/doc/About/GettingStarted.md:35]:
```suggestion

You may also use `next` to  use the latest pre-release version.

> Note: prior to the official 1.0 release of Bodiless, you *must* use at least `1.0.0-beta.11`.  Using just >`@bodiless/cli` without a version specifier will fail.

```

Comment (misterjones) [reply packages/bodiless-documentation/doc/About/GettingStarted.md:46]: ✅ Updated pre-release details in **Note** blocks.

**Review by wodenx — APPROVED** [review]

**Review by hvanyo — APPROVED** [review]

**Pull request merged**